### PR TITLE
Increment MH version + Update api response MHC

### PR DESCRIPTION
**Pull request by prankulmahajan**
The PR contains the following,

- [x] With new changes merged to main for MH in #7 , incremented the mount helper version in makefile to `0.0.3`
- [x] Follow same versioning in mount helper container, set the version to `0.0.2`
- [x] Fixed bug in mount-helper-container Makefile to require particular version of mount-helper in rpm build
- [x] Add mount output as part of API response in mount-helper-container
- [x] Update name of `stagingTargetPath` to `mountPath`
- [x] ~increment version tag in workflow (v0.0.5)~

-------------

TODO:
1. Decide how certs will be delivered from IKS use case -- Will be addressed via #10 
2. Fix MH UTs -- Will be addressed via #10 
3. Rebase this PR

### CODE REVIEW

**Review by anand-desai — APPROVED** [review]

**Review by saicharansunkara-ibm — APPROVED** [review]

**Comment by prankulmahajan** [comment]
@anand-desai @saicharansunkara-ibm Can you re-review the PR again and see if the convention looks good.

Once this PR is merged we need to finalise how certs issue and then can create a new release from workflow

**Comment by prankulmahajan** [comment]
@saicharansunkara-ibm Can you remove the do not merge label? Thanks

**Review by saicharansunkara-ibm — APPROVED** [review]

**Review by arahamad — APPROVED** [review]

**Review by anand-desai — APPROVED** [review]

**Pull request merged**
